Tools/styleui/styleui.py

```diff
@@ -126,30 +126,28 @@ def make_copy(args):
     ui_content = read_ui_file(args.ui_file_path)
```

Comment:
Honestly these python files that you converted from Python2 to Python3 can be deleted.
They were not being used as python 2 hasn't been supported for 4 year and keeping them around just adds more maintenance in the future.

Reply:
Will deprecate in a different PR, created a GHI: https://github.com/o3de/o3de/issues/17576